src/ImageSharp/Primitives/ComplexVector4.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace SixLabors.ImageSharp.Primitives
+{
+    /// <summary>
+    /// A vector with 4 values of type <see cref="Complex64"/>.
+    /// </summary>
+    internal struct ComplexVector4
```

Comment:
Implement `IEquatable<T>` etc  for best practice

Reply:
Done in https://github.com/SixLabors/ImageSharp/pull/842/commits/9491262b4b4214c273044910107f24b99ee65cd8.